venia-integration-tests/run-tests

```diff
@@ -17,5 +17,18 @@ PR_INSTANCE_URL=$1
 # collect update screenshots flag
 UPDATE_SCREENSHOTS=${2:-false}
 
-# run cypress tests and remove the container once done
-docker run --rm -v $PWD:/venia-integration-tests -w /venia-integration-tests --entrypoint=cypress cypress/included:8.3.1 run --browser chrome --config baseUrl=$PR_INSTANCE_URL,screenshotOnRunFailure=false --config-file cypress.config.json --env updateSnapshots=$UPDATE_SCREENSHOTS --headless
+# see if its a local instance or a pr instance
+PORT="$(echo $1 | sed -e 's,^.*:,:,g' -e 's,.*:\([0-9]*\).*,\1,g' -e 's,[^0-9],,g')"
```

Comment:
Seems a bit of a stretch to assume simply because it has a PORT number for it to be local. Can't we just add
`USE_DOCKER_NETWORK=${3:-false}`
or 
`IS_LOCAL=${3:-false}`

Reply:
This solution is built for apps built with buildpack which has a port while developing locally. You are right though buddy, I thought about adding a new arg since the second arg is already an optional arg I figured its easy to not ask for it 😅